static/options.interfaces.ts

```diff
@@ -51,12 +51,13 @@ export type LibsPerRemote = Record<string, LanguageLibs>;
 export type Options = {
     libs: Libs;
     remoteLibs: LibsPerRemote;
-    languages: Record<string, Language>;
```

Comment:
This is extra. Only the compileOptions addition is releated to the fix, but better typings are always good